examples/standalone-data-generator/model/App.py

```diff
@@ -1,23 +1,20 @@
-"""
-Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+from abc import abstractmethod, ABC
```

Comment:
Are they from standard python library?

Reply:
Yes, `abc` is python `Abstract Base Classes`, Reference link: https://docs.python.org/3/library/abc.html